standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -402,6 +402,8 @@ public enum ConfVars {
             "has an infinite lifetime."),
     CLIENT_SOCKET_TIMEOUT("metastore.client.socket.timeout", "hive.metastore.client.socket.timeout", 600,
             TimeUnit.SECONDS, "MetaStore Client socket timeout in seconds"),
+    CLIENT_CONNECTION_TIMEOUT("metastore.client.connection.timeout", "hive.metastore.client.connection.timeout", 10,
```

Comment:
Nice catch, if there is any concern I will keep the default connection timeout value same as socket timeout.

Reply:
@wecharyu, won't be socketTimeout = connectTimeout if not explicitly set?
{code}
public TSocket(TConfiguration config, String host, int port, int timeout)  {
    this(config, host, port, timeout, timeout);
  }
public TSocket(TConfiguration config, String host, int port, int socketTimeout, int connectTimeout)  {}
{code}

Reply:
Yes, I have changed the default connectionTimeout to be the same as socketTimeout. But IMHO it's not a good practice because if we set socketTimeout and not set connectionTimeout, the default long time will be gained for connection, I think we should change the behavior by default where socketTimeout = connectionTimeout.